src/components/global/nav-bar/nav-bar.vue

```diff
@@ -0,0 +1,103 @@
+<script>
+    import { defineComponent } from 'vue';
+
+    import MonteCinemaIcon from '../../../assets/icons/monte-cinema-logo.svg';
+    import ToggleNavOpenIcon from '../../../assets/icons/toggle-nav-open-icon.svg';
+    import ToggleNavClosedIcon from '../../../assets/icons/toggle-nav-closed-icon.svg';
+    import NavLinks from './nav-links.vue';
+    import NavButtonContainer from './nav-button-container.vue';
+
+    export default defineComponent({
+        name: 'NavBar',
+        components: {
+            MonteCinemaIcon,
+            ToggleNavOpenIcon,
+            ToggleNavClosedIcon,
+            NavLinks,
+            NavButtonContainer,
+        },
+        props: {
+            userType: {
+                type: String,
+                default: 'public', //Others: client, employee
+            },
+        },
+        data() {
+            return {
+                isScreenSmall: false,
+                isNavOpen: false,
+            }
+        },
+        methods: {
+            watchScreenSmall() {
+                this.isScreenSmall = window.innerWidth < 1024;
+            },
```

Comment:
Maybe we can avoid checking the window size using JavaScript and define CSS value which hides content on a given breakpoints using `display: none`?

CSS should be responsible for visual part as often as possible : )

Reply:
Ok, done!